src/components/layout/Footer.js

```diff
@@ -69,38 +69,70 @@ const Footer = ({ width }) => (
                 <Col md={6}>
                   <LinkList>
                     <Li>
-                      <Link to="/react/training">React Courses</Link>
+                      <Link to="/react/training" className="footer-site-links">
```

Comment:
@ric9176 we repeat the same class name in a few cases. Why don't we create an array like:
```
const footerLinks = [{
 to: "/react/training",
 txt: "React Courses"
}, etc]

```
then we simply do:
```
footerLinks.map(({to, txt})=> (< Li >
                      <Link to="/react/curriculum">React Curriculum</Link>	                      
                        to={to}
                        className="footer-site-links"
                      >
                       {txt}
                      </Link>
                    < / Li>	                    
      )
)
```

Reply:
:+1: